news/1750.bugfix

```diff
@@ -0,0 +1 @@
+Give Site Administrator permission to add users to groups. @wesleybl
```

Comment:
```suggestion
Fixed the permission check for adding users to groups and removing users from groups, so that it is allowed for users with the Site Administrator role. @wesleybl
```